src/Stratis.StratisD/Program.cs

```diff
@@ -30,7 +31,8 @@ public class Program
                     .UseMempool()
                     .UseWallet()
                     .AddPowPosMining()
-                    .UseApi()
+                    .UseApi()        
+                    .UseApps()
```

Comment:
I don't think we want to use apps in default Stratis Demon

Reply:
I think we do want to have Apps as a first class feature.
The idea is that soon we won't be looking at the console anymore but rather at a web page, so I don't mind this coming in now.

Reply:
Yeah I agree I would have it as part of the default deamon (same as api)

Reply:
Happy for it to be left out until the sec issues are resolved.

Reply:
Then it'll be in by default 💃

Reply:
It is the principle rather than current issues. I mean, it won't be ever solved, attack surface will always be greater as well as resource reqs.

Reply:
But this is a light wait project and nothing is added by default yet (infect I think we should load apps only on demand)
This will allow us to provide a set of default UIs to interact with the node without having to plug new apps.

Regarding security that's another topic, yeah the security issue  comes from the ability to plug new apps so if one does not plug any then there is not added risk.

Reply:
I'm not totally against it, like I would block it or something, just want to make sure everyone is on the same page about consequences of having it. I'd personally not include it, but if you are OK with all the consequences, then OK.

Reply:
What I mean is that I wouldn't include apps into the node at all, I would rather go with UI being separated application instead of browser. But it looks like this is too much conceptually different than what you are looking for.

Reply:
This could anyway be achieved if the separate UI would be in another processes using RPC to communicate with the node (which is similar to browser I guess).

Agree we need to carefully inspect what apps are delivered with the node itself. 
This is not different to the current swagger UI that is part of the node already.

Reply:
yeah, kind of hate swagger inside node :D